routemaster/subscription.rb

```diff
@@ -0,0 +1,22 @@
+module Routemaster
+  class Subscription
+
+    attr_reader :subscriber, :callback, :topics, :events
+
+    def initialize(options)
+      @subscriber = options.fetch('subscriber')
+      @callback   = options.fetch('callback')
+      @topics     = options.fetch('topics')
+      @events     = options.fetch('events').symbolize_keys
```

Comment:
huh, you're right but maybe we require it somewhere else? Because the method actually symbolizes the keys :thinking: